package.json

```diff
@@ -42,9 +42,7 @@
     "async": "2.4.0",
     "ramda": "0.23.0",
     "typescript": "2.3.2",
-    "tyranid": "0.1.57",
-    "mongodb": "2.1.14",
-    "monads.either": "0.2.0"
+    "tyranid": "0.1.57"
```

Comment:
More specific to `package.json` than this change (but can't add an inline comment for some reason). Think we need to either:
- Include the built files and adjust `main` to point to the compiled version, OR
- Have a `postinstall` task so that the consuming code will build the lib

I think option 2 is a little cleaner, but I know @bsouthga had issues with that route for `crosslead-adapters`. Ben, do you remember what the issue was?